Dockerfile

```diff
@@ -1,5 +1,10 @@
 FROM openjdk:17
+WORKDIR /prepare
+COPY src/main Manifest.txt ./
+COPY data ./data
+RUN javac -d . Solution.java && jar cfm Solution.jar Manifest.txt main/*.class
+
+FROM openjdk:17-alpine
 WORKDIR /app
-COPY ./src/main .
-RUN javac -d . ./*
-ENTRYPOINT ["java", "-classpath", "./", "main.Solution"]
\ No newline at end of file
+COPY --from=0 /prepare/ ./
+ENTRYPOINT ["java", "-jar", "Solution.jar"]
```

Comment:
ENTRYPOINT :+1: 
https://stackoverflow.com/questions/21553353/what-is-the-difference-between-cmd-and-entrypoint-in-a-dockerfile